### PR TITLE
fixing test check for jenkins

### PR DESCRIPTION
**Pull request by winsvega**

### CODE REVIEW

File: test/libethereum/BlockChain.cpp [hunk 1]
```diff
@@ -38,6 +38,7 @@ BOOST_AUTO_TEST_SUITE(BlockChainSuite)
 
 BOOST_AUTO_TEST_CASE(output)
 {
+	TestOutputHelper::initTest();
```
Comment (chriseth) [inline test/libethereum/BlockChain.cpp:41]: For the next time, it is probably better to create a fixture here.

**Comment by chriseth** [comment]
Gas pricer still fails. Is that an issue of the tests?

**Comment by winsvega** [comment]
I don't think so. windows tests are fine for whatever reason. 
it seems that gaspricer test has a diffrent result depend on a platform for frontier network

**Comment by CJentzsch** [comment]
@winsvega As mentioned before, I would suggest making the GasPricer checks range based checks. Those are effectively floats represented as integer. It would make more sense ff the return value is checked against a range instead a fixed number.

**Comment by chriseth** [comment]
Ah, I did not realise that fact. Christoph is right, we should not compare them at unit precision. This might create a lot of confusion if the tests are run on different platforms.

**Comment by winsvega** [comment]
could we merge this and then open a PR for range checks?

**Pull request merged**